service/routes.py

```diff
@@ -70,16 +70,24 @@ def list_shopcarts():
 # -----------------------------------------------------------
 
 
-@app.route("/shopcarts/<customer_id>", methods=["POST"])
-def add_shopcart(customer_id):
+@app.route("/shopcarts", methods=["POST"])
```

Comment:
I think you need to change the comment and logger. Otherwise, it will confuse.

Reply:
youre right, will change that